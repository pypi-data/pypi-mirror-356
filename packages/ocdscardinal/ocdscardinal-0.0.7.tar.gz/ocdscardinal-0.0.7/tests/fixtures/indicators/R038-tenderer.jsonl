{"ocid":"0","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"D"}]},{"status":"valid","tenderers":[{"id":"0"}]},{"status":"valid","tenderers":[{"id":"1"}]},{"status":"valid","tenderers":[{"id":"2"}]},{"status":"valid","tenderers":[{"id":"3"}]}]}}
{"ocid":"1","bids":{"details":[{"status":"valid","tenderers":[{"id":"D"}]}]}}
