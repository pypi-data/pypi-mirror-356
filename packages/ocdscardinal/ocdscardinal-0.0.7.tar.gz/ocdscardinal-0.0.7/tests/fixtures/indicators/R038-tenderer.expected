{"Tenderer":{"D":{"R038":0.5}}}
